bucket/qbittorrent.json

```diff
@@ -18,13 +18,25 @@
     },
     "pre_install": [
         "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
-        "if ( (!(Test-Path \"$persist_dir\\profile\")) -and (Test-Path \"$env:APPDATA\\qBittorrent\") ) {",
-        "    Write-Host \"Scoop is migrating qbittorrent to use portable mode by default.\" -ForeGroundColor Yellow",
-        "    Write-Host \"For details, see: https://github.com/ScoopInstaller/Extras/pull/7173\" -ForeGroundColor Yellow",
-        "    Write-Host \"Copying files from APPDATA to Scoop directory...\" -ForeGroundColor Yellow",
-        "    New-Item \"$persist_dir\\profile\\config\", \"$persist_dir\\profile\\data\" -ItemType Directory | Out-Null",
-        "    Copy-Item \"$env:APPDATA\\qBittorrent\\*\" \"$persist_dir\\profile\\config\" -Force -Recurse | Out-Null",
-        "    Copy-Item \"$env:LOCALAPPDATA\\qBittorrent\\*\" \"$persist_dir\\profile\\data\" -Force -Recurse | Out-Null",
+        "if ((!(Test-Path \"$persist_dir\\profile\")) -and ((Test-Path \"$scoopdir\\persist\\qbittorrent-portable\\profile\") -or (Test-Path \"$env:APPDATA\\qBittorrent\"))) {",
+        "    Write-Host \"Scoop is migrating qbittorrent to use portable mode by default.\" -ForegroundColor Yellow",
+        "    Write-Host \"For details, see: https://github.com/ScoopInstaller/Extras/issues/5845\" -ForegroundColor Yellow",
+        "    if ((Test-Path \"$scoopdir\\persist\\qbittorrent-portable\\profile\") -and (Test-Path \"$env:APPDATA\\qBittorrent\")) {",
```

Comment:
Why is persist_dir not being used here?

Reply:
Maybe persist_dir/../qbittorrent-portable

In case Scoop adds support for changing the location of persist_dir?